packages/aragon-api-react/README.md

```diff
@@ -108,6 +108,36 @@ function App() {
 }
 ```
 
+#### `apps`
+
+The complete list of apps installed in the organization. Its value is an empty array (`[]`) until
+the list of apps are loaded.
+
+Each object in the array holds the following keys:
+
+- `appAddress`: the app's contract address
```

Comment:
@sohkai Is the idea to progressively move away from `proxyAddress` / `contractAddress`? And shouldn’t we pass `currentApp` through `transformAppInformation(app)` too?

Reply:
> And shouldn’t we pass currentApp through transformAppInformation(app) too?

I think we do this: https://github.com/aragon/aragon.js/blob/master/packages/aragon-wrapper/src/rpc/handlers/get-apps.js#L53

> Is the idea to progressively move away from proxyAddress / contractAddress?

I'd like to. `proxyAddress` has always been a bit of a misnomer, and confusing. `contractAddress` even more so. The important thing, outside of aragonOS (and some tiny parts in aragon.js), is that each app has a contract address. Maybe it's a proxy, maybe it has a base implementation, but these details shouldn't matter to any developer unless they're building something custom on the contract level (at which point aragon.js probably isn't the library for them).

Reply:
> I think we do this: https://github.com/aragon/aragon.js/blob/master/packages/aragon-wrapper/src/rpc/handlers/get-apps.js#L53

Oh I missed that :+1: 

> I'd like to. proxyAddress has always been a bit of a misnomer, and confusing. contractAddress even more so. The important thing, outside of aragonOS (and some tiny parts in aragon.js), is that each app has a contract address. Maybe it's a proxy, maybe it has a base implementation, but these details shouldn't matter to any developer unless they're building something custom on the contract level (at which point aragon.js probably isn't the library for them).

That sounds good yes! I wonder if we should also move to this naming in aragon/aragon, just for consistency.

Reply:
Haha absolutely... we can start by mapping the values over to a different schema in aragonjs-wrapper.

It's a bit hard to update `@aragon/wrapper` because the CLI also depends on it, but I hope with the refactor they're doing they'll be able to accommodate for breaking changes more easily.